src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -135,10 +135,10 @@ public static class StandardTrainersCatalog
         /// <param name="labelColumnName">The name of the label column.</param>
         /// <param name="featureColumnName">The name of the feature column.</param>
         /// <param name="exampleWeightColumnName">The name of the example weight column (optional).</param>
-        /// <param name="l2Regularization">The L2 <a href='tmpurl_regularization'>regularization</a> hyperparameter.</param>
-        /// <param name="l1Threshold">The L1 <a href='tmpurl_regularization'>regularization</a> hyperparameter. Higher values will tend to lead to more sparse model.</param>
```

Comment:
><a href='tmpurl_regularization'> [](start = 45, length = 32)

Just another note that we will want to point to a standard doc for L1 etc. and not put details here.

Reply:
Although `Threshold` => `Regularization` is great!

---
In reply to: [267521149](https://github.com/dotnet/machinelearning/pull/3029#discussion_r267521149) [](ancestors = 267521149)